NewForce-Cohort-9/tabloidmvc-stafford-infection-1
# View Comments on Post

# Description

Added functionality View Comments that allows you to view comments for a specific post. This allows you to see a list of all the Comments on a Post so that you can read and take part in the discussion on a particular Post.
Trello Ticket: https://trello.com/b/LapTOJIT/stafford-infection

## Type of change

# Testing Instructions

1. Ensuring that you have cloned down the repo and added the database, git fetch --all in the master branch to retrieve my branch named fw-view-comments-ticket19 
2. git checkout fw-view-comments-ticket19 to switch to my branch. If you don't see the most recent version of my files, git pull origin fw-view-comments-ticket19
3. Open up Visual Studio. 
4. Because Create New comment functionality is not yet in place, create a new query on the TabloidMVC database and execute the following query to add a few comments to the comment table (which is now still empty) in the database: 
SET IDENTITY_INSERT [Comment] ON;

INSERT INTO [Comment] ([Id], [PostId], [UserProfileId], [Subject], [Content], [CreateDateTime]) 
VALUES 
(1, 1, 1, 'C# is the best', 'I do agree', SYSDATETIME());

INSERT INTO [Comment] ([Id], [PostId], [UserProfileId], [Subject], [Content], [CreateDateTime]) 
VALUES 
(2, 1, 1, 'C# is the best', 'But perhaps it’s too soon to tell.', SYSDATETIME());

INSERT INTO [Comment] ([Id], [PostId], [UserProfileId], [Subject], [Content], [CreateDateTime]) 
VALUES 
(3, 2, 1, 'Javascript for Beginners', 'It really is a good one for beginners.', SYSDATETIME());

INSERT INTO [Comment] ([Id], [PostId], [UserProfileId], [Subject], [Content], [CreateDateTime]) 
VALUES 
(4, 2, 1, 'Javascript for Beginners', 'You should try it.', SYSDATETIME());

SET IDENTITY_INSERT [Comment] OFF;

6.To view the changes in the Comment table, execute the following query:
SELECT* FROM Comment
7. Run the application in a browser window
8. You should see the 'Home' page with a navigation bar at the top of the page. Select 'Login' and log in with this email: [admin@example.com]
9. Go to Posts on the nav bar
10. You will see an “eye” icon/button, click on it to go to Post Details 
11. Given you are viewing the Details of a Post, when you select the View Comments button
12. You are directed to the Comments list page for the Post
13. Each Comment displays the following information: Subject, Content, Author's Display Name and Creation date (MM/DD/YYYY)
14. The list is in order of creation date with the most recent on top
15. The title of the related Post is displayed at the top of the page
16. There is a link back to the Post when you click on the title of the Post

# Checklist:
